### DISCUSSION TITLE
[Question][Hw1]

### DISCUSSION THREAD

**Discussion by reinattwijaya** [prosyslab-classroom/cs348-information-security#673] [Q&A]
Name: Reinatt Hansel Wijaya

what is the time limit for each task?

**Comment by KalasLavas** [comment]
Technically [40 minutes](https://docs.google.com/document/d/1FWXTf4LI-DMvZIu2aRPnzGOzC7flR_G_8Bf9ZDGjX7s/edit), but your score will get lower if it runs for too long #234 . From my experience ~1 sec per test case should be fine.

**Comment by reinattwijaya** [comment]
I see, that's why, thanks alot
